.conan/profiles/config/debug

```diff
@@ -1,2 +1,4 @@
 [settings]
-build_type=Debug
\ No newline at end of file
+build_type=Debug
+mold*:build_type=Release
```

Comment:
I think we can probably move duplicted setting into a common profile file and use python code to copy the `build_type` into the setting (accounting for mapping RelWithDebInfo to Release).

We do something similar with the OS settings: https://github.com/Twon/Morpheus/blob/fa14cd20929517cf2d200065715e7ae25762003f/.conan/profiles/os/current#L2

Reply:
Good point. We could create a new profile file `.conan/profiles/packages/settings` where with specific profile rendering code we can add settings for specific packages. Now we want to specialise the build process for Mold linker but we can eventually reuse this logic in the future also for other packages that need specific settings. We are going to include the new profile file in all the build profiles for `clang/gcc/msvc`. In this way maybe we are gaining flexibility for future needs. What do you think?

Reply:
Done, I added a new profile file.